openapi/paths/payment-methods.yaml

```
@@ -1,26 +1,26 @@
 get:
-  description: ''
-  operationId: ListPaymentMethods
+  description: Get information about a payment method.
```

When do we phrase descriptions as 'Get information about' and when do we phrase descriptions as 'Get details about'?
```suggestion
  description: List activated payment methods, optionally filtered by query parameters.
```